src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/AddKeepAlivesStep.cs

```diff
@@ -11,6 +11,9 @@
 namespace MonoDroid.Tuner
 {
 	public class AddKeepAlivesStep : BaseStep
+#if !ILLINK
+		, IAssemblyModifierPipelineStep
+#endif  // !ILLINK
```

Comment:
If `src\Microsoft.Android.Sdk.ILLink\Microsoft.Android.Sdk.ILLink.csproj` had `IAssemblyModifierPipelineStep.cs` we could drop the `#if`?